lib/game_info.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Info
```

Comment:
You don't need this module. Replace `welcome_info` with `puts I18n.t(:greeting)` in your code